contracts/DxDevDependencies.sol

```diff
@@ -9,7 +9,7 @@ pragma solidity ^0.4.21;
  
  
 // TODO: Use the same getter pattern also for dependencies
-import "@gnosis.pm/util-contracts/contracts/StandardToken.sol";
+import "@gnosis.pm/util-contracts/contracts/GnosisStandardToken.sol";
```

Comment:
I think there was a consensus within gnosis that we would like to use the open-zeppelin contracts, if available. here we could use their standardtoken

Reply:
Yes, that's our idea too, only thing in this PR we didn't want to change the code. 
We have other pending PR related to som changes in the DutchX, but we want to go one by one :) 

This change is just to avoid using StandardToken as the name, because is causing lot's of problems because of name collisions.